include/seqan3/alignment/configuration/align_config_result.hpp

```diff
@@ -107,10 +155,14 @@ class result : public pipeable_config_element<result<with_type>, with_type>
  * \{
  */
 //!\brief The default constructor defaults to score-only computation.
-result() -> result<detail::with_score_type>;
+result() -> result<detail::with_score_type, int32_t>;
```

Comment:
I think for this and the next deduction guide, you don't need to add `int32_t` since it is a defaulted parameter...

Reply:
Good point. I think I don't need this type deduction at all... I'll remove it